physionet-django/notification/utility.py

```diff
@@ -1025,3 +1025,19 @@ def notify_event_participant_application(request, user, registered_user, event):
     body = loader.render_to_string('events/email/event_registration.html', context)
     # Not resend the email if there was an integrity error
     send_mail(subject, body, settings.DEFAULT_FROM_EMAIL, [user.email], fail_silently=False)
+
+
+def notify_users_of_training_expiry(user, training, expiry):
+    """
+    Send the training expiry email.
+    """
+
+    subject = f"{settings.SITE_NAME} Training Expiry Notification"
+    context = {
+        'name': user.get_full_name(),
+        'SITE_NAME': settings.SITE_NAME,
+        'training': training,
+        'expiry': expiry
+    }
+    body = loader.render_to_string('training/email/training_expiry_notification.html', context)
+    send_mail(subject, body, settings.DEFAULT_FROM_EMAIL, [user, ], fail_silently=False)
```

Comment:
let's change this to a user-specific banner, at the beginning just on the training page

Reply:
Need some more clarification on the user-specific banner on the training page.

Reply:
I will be adding a conditional message displayed on the training page when training will expire in the next 30 days.